### PR TITLE
Add DigitalOcean DNS template

### PR DESCRIPTION
**Pull request by janeczku**
ready to merge

### CODE REVIEW

File: infra-templates/digitalocean-dns/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cloudflare:
```
Comment (galal-hussein) [inline infra-templates/digitalocean-dns/0/docker-compose.yml]: @janeczku I think this is a typo

File: infra-templates/digitalocean-dns/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      label: "DNS Name Template"
+      description: |
+        Name template used to construct the subdomain part (left of the domain) of the DNS record names.
+        Supported placeholders: %{{service_name}}, %{{stack_name}}, %{{environment_name}}.
+        By default DNS entries will be named '<service>.<stack>.<environment>.<domain>'.
+      type: "string"
+      default: "%{{service_name}}.%{{stack_name}}.%{{environment_name}}"
+      required: false
+
+cloudflare:
```
Comment (galal-hussein) [inline infra-templates/digitalocean-dns/0/rancher-compose.yml]: same here

**Comment by galal-hussein** [comment]
@janeczku Other than these minor changes, i tested the template and it works with DigitalOcean as expected.

**Comment by janeczku** [comment]
@galal-hussein fixed typos

**Comment by galal-hussein** [comment]
@janeczku Thanks...LGTM

**Pull request merged**